cypress/e2e/profile.cy.js

```diff
@@ -52,4 +52,19 @@ describe('Profile', () => {
 
         cy.get('.Users__Edit__content').screenshot('application/profile/edit/api-keys/form', { padding: [0, 0, 0, 330] })
     })
+
+    it.only('About', () =>{
+        cy.loginAs('admin')
+        cy.visitApp('/dashboard')
+
+        cy.wait(1000)
```

Comment:
Jinak wait by se takhle většinou používat neměl, viz: https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting

Jaký to má tady účel?

Reply:
Ok, dělala jsem podle "Change language", kde je to takto použito. Tam je to teda taky špatně?

Reply:
Nejspíš ano.

Jde o to, že cypress automaticky čeká, dokud tam nenajde ten element přes get. Tzn., že na tom dalším řádku `cy.get('#menu_profile')` se bude čekat, dokud tam nebude `'#menu_profile`. Někdy může být potřeba trochu upravit, na co čekat, tím jak se tam různé části donačítají dynamicky.

Pokud by se používal nějaký random čas, tak se buď bude čekat zbytečně dlouho v případě, že to zrovna poběží rychleji nebo to naopak nebude stačit, když to bude trvat moc dlouho. Takhle se prostě počká jen tak dlouho, než tam ten další element bude.